### DISCUSSION TITLE
Multiple/recovery emails for account

### DISCUSSION THREAD

**Discussion by bryan-brancotte** [kjolley/BIGSdb#782] [Ideas]
Dear Keith

In order to keep long term contact with accounts sometime created by phd students, we (@fedex88, Sylvain and I) are wondering if there is any way to have multiple email. @fedex88 indicates me that they tried multiple emails separated with a `;` in the email field but then automated notification stop working. Any hint ? Any think in the roadmap ?

Thanks for your time

Bryan

**Comment by kjolley** [comment]
There isn't currently a way of doing this. I think to store multiple emails, the best solution would be to change the email field in the users table to a text array, rather than trying to separate them within a single value. This would entail changing the table in the seqdef, isolate and site-wide users databases. Any code that uses the field will need to be updated, so there is potentially quite a lot to change - isolate info, submission system, scripts etc.

There is also the issue that site-wide user accounts require E-mails to be unique so that people don't sign up multiple times, either by mistake or in order to get around the pending submission limits. So, an account would still require the primary email to be unique.

I wonder if there is a better way altogether. We already have user groups (used for group permissions) - perhaps we could have something similar to group accounts by research group. This could have a description and link to a PI user account.

I need to give this some more thought.

**Comment by bryan-brancotte** [comment]
Thanks for clarifying that `;` in email is not the right way to do it.

This issue is *not* a feature request, it is more a use case that we wanted to submit to you in case you had any feature already there.
